### ISSUE TITLE
Broken link to Amethyst CLI

### ISSUE THREAD

**Issue by golergka** [amethyst/tools#75]
README has a broken link: https://github.com/amethyst/tools/tree/master/src/cli

**Comment by BradLugo** [comment]
**TL;DR**: Should we remove the **Toolchain** section altogether or just get rid of the link?

It seems like the documentation and the implementation for `amethyst/tools` are inconsistent with each other. Looking at the documentation and older commits it appears the CLI was one of many tools intended to be built and tracked in this repo. Looking at the current implementation though it seems like the CLI is the _only_ tool designed for this repo. This change to the single CLI tool was merged in due to #39; however, there's no explanation for the rewrite. This broken link is a product of that PR. I'd like to hear opinions from the maintainers on the direction they'd like to go, but I think either the repo needs to be renamed to reflect the single tool, CLI nature of this project (e.g., `amethyst/cli`) or we revert the rewrite from #39 (particularly the changes that moved the code out of `src/cli` into just `src`).

**Comment by torkleyy** [comment]
Yes, this repo should probably be renamed to `amethyst-cli`.
